### ISSUE TITLE
File moving and renaming

### ISSUE THREAD

**Issue by ChaosTherum** [xbapps/xbvr#296]
Would it be possible to add the ability to move and rename files based on templates. I would even be happy if you decided on a format and forced it. Right now the process is search for the actors, scene name, resolution, format, etc. What would be real nice is being able to just drop my files in then confirm a match then be able to click a button to move and rename files into a decently organized flat structure. I know part of this software is generating nice file structures for players through dlna but it's also nice to just have some basic information in the file names themselves.

**Comment by cld9x** [comment]
Unfortunately that's one of the things I didn't want to be doing right from the start, which was one of the decisions that shaped xbvr the way it is right now (i.e. filename being one of the least important pieces of metadata).

Let's revisit this one when we stabilize the codebase.

**Comment by ChaosTherum** [comment]
Yeah no problem with that I was just putting the idea out there since I didn't see any issues for it. It would definitely be a really nice addition.

**Comment by HotBlack77** [comment]
Is there a chance this will be revisited one day? 
My use case is that XBVR does a much better job of tagging VR content and through careful renaming I can then "sync" with stash by scraping the now better named files.
